lua/damagelogs/shared/lang/brazilian_portuguese.lua

```diff
@@ -249,15 +248,13 @@ DamagelogLang.portugues = {
     AllPlayers = "Todos os jogadores",
     InvalidPlayerChat = "Jogador inválido!",
     CannotLeaveChat = "Você não pode sair como o único admin!",
-    Send = "Enviar",
     AdminsDisconnectedChat = "Todos os admins desconectaram! O chat foi encerrado.",
     ChatClosedBy = "Este chat foi fechado por %s.",
     AdminLeaveChat = "%s saiu do chat.",
     AllPlayersShort = "todos os jogadores",
     ForcedNotification = "%s forçou %s a permanecer no chat",
     ReleasedNotification = "%s liberou %s do chat",
     ReportHistory = "Histórico do report %u",
-    ReopenChat = "Reabrir este chat",
```

Comment:
This is used in the chat
https://github.com/BadgerCode/tttdamagelogs/blob/master/lua/damagelogs/client/chat.lua#L857

Reply:
This is also in line 69